hazelcast/src/main/java/com/hazelcast/nio/tcp/WriteHandler.java

```diff
@@ -112,12 +181,15 @@ public void offer(SocketWritable packet) {
         schedule();
     }
 
+
     private SocketWritable poll() {
         for (; ; ) {
+            boolean urgent = false;
             SocketWritable packet = urgentWriteQueue.poll();
 
             if (packet == null) {
                 packet = writeQueue.poll();
+                urgent = true;
```

Comment:
As I already mentioned at `apple-jcache` slack group, since it is taken from `writeQueue` it is not urgent. It should be the opposite.